internal/pubsub/pubsub.go

```diff
@@ -288,15 +288,15 @@ func (s *Server) NumClientSubscriptions(clientID string) int {
 
 // Publish publishes the given message. An error will be returned to the caller
 // if the context is canceled.
```

Comment:
```suggestion
// if the pubsub server has shut down.
```

As a follow-up I'd suggest we maybe get rid of the `error` return entirely. Rationale: t's not really actionable for the caller, since pubsub isn't the lifecycle regulator for the services that depend on it. Now that we don't report failures for the publish itself, I feel like we don't need this.